src/openzaak/components/zaken/api/viewsets.py

```diff
@@ -2070,14 +2071,9 @@ def perform_create(self, serializer):
         )
 
 
-@extend_schema(
-    summary="Short een zaak op",
-    description=mark_experimental("Schort een zaak op en maak een status aan."),
-)
-class ZaakOpschortenViewset(
+class ZaakUpdateActionViewSet(
```

Comment:
I seem to be getting this error when loading the redoc page:

```
Warning: operationId "None_post" has collisions [('/zaak_bijwerken/{uuid}', 'post'), ('/zaak_opschorten/{uuid}', 'post'), ('/zaak_verlengen/{uuid}', 'post')]. resolving with numeral suffixes.
```

Could you make sure they have an operationId?

Reply:
fixed by changing to  `extend_schema_view`